shell/common/engine.h

```diff
@@ -127,6 +127,8 @@ class Engine final : public RuntimeDelegate {
   // |RuntimeDelegate|
   FontCollection& GetFontCollection() override;
 
+  Animator* GetAnimator() { return animator_.get(); }
```

Comment:
Why's that?

Reply:
Because someone could hold that raw pointer to do something dangerous such as: 1. delete that pointer; dereference that pointer when the unique_ptr is destructed in the future. I think we try to minimize the usage of raw pointer as much as possible, especially in public interfaces.